.github/workflows/reusable.yml

```diff
@@ -72,7 +72,6 @@ jobs:
         docker build -t disc-dev-${{ inputs.name }} \
           --build-arg BASEIMAGE=${{ inputs.base_image }} \
           ${{ inputs.extra_build_args }} \
-          ${{ inputs.extra_envs }} \
```

Comment:
why removed..

Reply:
+1

Reply:
`docker build` does not have  `-e` option